website/templates/website/base.html

```diff
@@ -1,22 +1,40 @@
 {% load static%}
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <title>Lazy Guide</title>
+    <link rel="stylesheet" type="text/css" href="{% static 'website/styles.css' %}">
```

Comment:
```suggestion
    <link rel="stylesheet" type="text/stylesheet" href="{% static 'website/styles.css' %}">
```

It's taking the css after this change, please try and let me know.